src/state_machine.zig

```diff
@@ -1270,32 +1260,31 @@ test "create/lookup/rollback accounts" {
                 .flags = .{
                     .debits_must_not_exceed_credits = true,
                 },
-                .debits_pending = math.maxInt(u64),
-                .debits_posted = 60,
-                .credits_pending = math.maxInt(u64),
-                .credits_posted = 80,
+                .debits_pending = 1,
+                .debits_posted = 1,
+                .credits_pending = 1,
+                .credits_posted = 1,
```

Comment:
Here too, we can set to 0 the balances that should only be checked after `debits_pending`, to enforce consistent order of state machine logic.